pipeline/main.yaml

```diff
@@ -0,0 +1,76 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Main template for a CI/CD Serverless application."
+Parameters:
+  AppName:
+    Type: String
+    Description: Name of the application
+    MinLength: "1"
+    MaxLength: "80"
+    AllowedPattern: "[A-Za-z0-9-]+"
+    ConstraintDescription: Malformed input parameter. AppName must only contain upper and lower case letters, numbers, and -.
+  SAMInputFile:
+    Type: String
+    Description: The filename for the SAM file.
+    Default: slack-anonymous.yml
+  SAMOutputFile:
+    Type: String
+    Description: The filename for the output SAM file from the buildspec file.
+    Default: post-saml.yaml
+  SlackWebHookUrl:
+    Type: String
+    Description: Slack WebHook URl
+  SlackChannel:
+    Type: String
+    Description: Slack channel
+    Default: "testing1"
+  CodeBuildImage:
+    Type: String
+    Default: "aws/codebuild/nodejs:4.3.2"
+    Description: Image used for CodeBuild project.
+  CICDS3BucketName:
+    Type: String
+    Default: "anon-cicd"
+    Description: S3 Bucket where the pipeline templates live
+  GitHubRepoName:
+    Type: String
+    Description: The GitHub repo name
+  GitHubRepoBranch:
+    Type: String
+    Description: The GitHub repo branch code pipelines should watch for changes on
+    Default: master
+  GitHubUser:
```

Comment:
This is a little confusing when it's called `user`. CodePipeline will use this to determine the repo path.
eg: `unicornpartners/anonna-serverless` is constructed as `[owner]/[repo]`

I suggest calling this out in the description text.